### ISSUE TITLE
Improve UI

### ISSUE THREAD

**Issue by zorroblue** [metakgp/chitragupta#3]
Currently the [page](https://wiki.metakgp.org/w/MetaKGP_Github_Contributors) is a simple list of contributors.
We could improve the display in a more elegant manner, possibly in a tabular format with a column also showing the list of repos he/she has contributed to.

Discussion on possible ideas is highly recommended before submitting the PR. :)

**Comment by icyflame** [comment]
@zorroblue This is a valid issue but you should elaborate on the _kind_ of changes you are expecting in a patch addressing this. The current title is very generic and unhelpful. :slightly_smiling_face:

**Comment by zorroblue** [comment]
@icyflame Yeah, I agree. I can think of using a table to display, but I felt if the contributor could pitch in a better idea, that would be better than me giving an easier solution. That's why I am keeping this open to discussion.

I'll anyways add the bare minimum improvement idea I can think of, though. Thanks :+1:

**Comment by icyflame** [comment]
Sure, definitely take contributions from the contributor. It's just that sometimes I find it hard to initiate a conversation around an idea I have, on a project I am new to. :slightly_smiling_face:

**Comment by aditodkar** [comment]
Can I work on this issue? I am familiar with python, HTML, CSS, JS.

**Comment by zorroblue** [comment]
Sure @aditodkar Have a look at the pywikibot API and discuss your ideas on this thread.
